tests/test_zeitwochenmarkt.py

```diff
@@ -36,14 +39,13 @@ def test_image(self):
     def test_ingredients(self):
         self.assertEqual(
             [
-                '1 Kugel "Fior di Latte" (oder eine Burrata)',
-                "1 Kohlrabi (klein)",
-                "1 Fenchelknolle",
-                "1/2 Bund Basilikum",
-                "2 EL Pistazienkerne",
-                "4 EL Olivenöl",
-                "2 EL Zitronensaft",
-                "1 TL Honig",
+                "Kohlrabi",
+                "Fenchelknolle",
+                "Basilikum",
+                "Pistazienkerne",
+                "Olivenöl",
+                "Zitronensaft",
+                "Honig",
```

Comment:
Mostly looks good - should be ready to merge soon!

Something's up with the ingredients list for `zeitwochenmarkt` here though - the quantities have gone missing?

Reply:
Fixed, not sure what happened with this one. Required a scraper rework.

Reply:
No problem - was the HTML also updated for this one?  (I don't see any `testhtml` updates for it)

Reply:
Oddly it wasn't updated in this batch, i just added a commit with a new html for this site specifically.